test/e2e/support/utils.js

```diff
@@ -136,6 +136,12 @@ export async function newPost( { postType, enableTips = false } = {} ) {
 	}
 }
 
+export async function disablePrePublishChecks( ) {
+	await page.click( '.edit-post-more-menu' );
+	await page.waitForSelector( '.components-popover__content' );
+	await page.click( '.components-popover__content > .components-menu-group:nth-child(3) button:nth-child(3)' );
```

Comment:
This is a bit of a fragile selector, is there anything better/better-named we could use to select the "Don't show this again" checkbox? It looks like `. editor-post-publish-panel__footer input[type=checkbox]` would work assuming `CheckboxControl` outputs an actual checkbox.

Reply:
Solved at d7c7151

I was reluctant to introduce a specific selector _just for testing_ but, on a second thought, it's totally worth it in this case. Note also that the code uses the general menu, not the actual checkbox within the publish sidebar. We can't guarantee the publish sidebar is enabled unless we use the general menu.